Formula/lib/librime.rb

```diff
@@ -0,0 +1,71 @@
+class Librime < Formula
+  desc "Rime Input Method Engine"
+  homepage "https://rime.im"
+  url "https://github.com/rime/librime.git",
+    tag:      "1.12.0",
+    revision: "c7ab6390c143a11d670f3add41218111edb883c9"
```

Comment:
nit comment to conform with formulae style

```suggestion
      tag:      "1.12.0",
      revision: "c7ab6390c143a11d670f3add41218111edb883c9"
```